sdk/python/testvectors/README.md

```diff
@@ -1,5 +1,3 @@
 # testvectors
 
 This directory contains testvectors generator written from scratch based on pre-existing testvectors.
```

Comment:
not sure if we need this readme?

Reply:
we probably should have readme for any tool/script (or at least describe usage _somewhere_)